app.js

```diff
@@ -8,8 +12,20 @@ app.get("/api/topics", getTopics);
 
 app.get("/api", getEndPoints);
 
+app.get("/api/articles/:article_id", getArticleById);
+
 app.all("*", (req, res) => {
   res.status(404).send({ msg: "Endpoint not found!" });
 });
 
+app.use((err, req, res, next) => {
+  if (err.status === 404) {
+    res.status(404).send(err);
```

Comment:
Again, this is fine at the moment, but have a look at the error handling notes as to how we can make this reusable for custom errors that are being rejected from the model. If we have more than one later on it'll just make it a little easier and less code to write 👍